src/components/Modal/ModalBasic/ModalShareBtns.tsx

```diff
@@ -0,0 +1,25 @@
+import FIcon from "@/assets/svg/FIcon";
+import KakaoIcon from "@/assets/svg/KakaoIcon";
+import LinkIcon from "@/assets/svg/LinkIcon";
+import styles from "./ModalBasic.module.scss";
+
+function ModalShareBtns() {
```

Comment:
요렇게 표현해주면 더 좋을 것 같아요~
```suggestion
function ModalShareButtons() {
```